packages/runtime/lib/logger.js

```diff
@@ -5,19 +5,100 @@ import pretty from 'pino-pretty'
 
 export { abstractLogger } from '@platformatic/foundation'
 
-const customPrettifiers = {
-  name (name, _, obj) {
-    if (typeof obj.worker !== 'undefined') {
-      name += ':' + obj.worker
-      obj.worker = undefined // Do not show the worker in a separate line
-    }
+// Extracted from https://github.com/debug-js/debug/blob/master/src/node.js
+const colors = [
+  20, 21, 26, 27, 32, 33, 38, 39, 40, 41, 42, 43, 44, 45, 56, 57, 62, 63, 68, 69, 74, 75, 76, 77, 78, 79, 80, 81, 92,
+  93, 98, 99, 112, 113, 128, 129, 134, 135, 148, 149, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172,
+  173, 178, 179, 184, 185, 196, 197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 214, 215, 220, 221
+]
+
+function createLoggerContext () {
+  const context = {
+    colors: {},
+    maxLength: 0,
+    updatePrefixes (ids) {
+      context.colors = {}
+      context.maxLength = 0
+
+      for (const id of ids) {
+        context.maxLength = Math.max(context.maxLength, id.length)
+        let hash = 0
+
+        if (!pretty.isColorSupported) {
+          context.colors[id] = ''
+          continue
+        }
+
+        // Calculate the hash of the id to pick a color
+        for (const char of id) {
```

Comment:
I'd avoid this way to set color, that is kind of a random, some similar colors could be selected.
I'd use a sequence with very different colors, for example https://d3js.org/d3-scale-chromatic/categorical
I may also want to specify a color for an application